scripts/release.sh

```
@@ -0,0 +1,67 @@
+      sleep 1
+    else
+      break
+    fi
+  done
+
+  if ! pg_isready -d $POSTGRES_DSN &> /dev/null; then
+    pg_isready -d $POSTGRES_DSN
+    exit $?
+  fi
+
+  for SCRIPT in $(ls -1 $1); do
```

Shellcheck (0.10.0)

[warning] Iterating over ls output is fragile. Use globs.

(SC2045)